checker/group_param_types_checker.go

```diff
@@ -84,6 +90,27 @@ func typeName(expr ast.Expr) string {
 		return result + " " + typeName(expr.Value)
 	case *ast.Ellipsis:
 		return "..." + typeName(expr.Elt)
+	case *ast.MapType:
+		return fmt.Sprintf("map[%s]%s", typeName(expr.Key), typeName(expr.Value))
+	case *ast.ArrayType:
+		return fmt.Sprintf("[%s]%s", typeName(expr.Len), typeName(expr.Elt))
+	case *ast.FuncType:
+		var paramTypes []string
```

Comment:
Maybe we can move this case to a separate function?

Reply:
Done.